### ISSUE TITLE
Missing & multiple libs

### ISSUE THREAD

**Issue by maristhek** [LilyGO/TTGO-T-Beam#44]
Cannot find axp20x.h
Compilation terminates with 'multiple libraries found for WiFi.h'

Another waste of time.

**Comment by no1call** [comment]
Yeah a lot of stuff missing.....most examples don't work because of missing libraries...i guess someone is making money selling boards.....
